docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+services:
+  app:
+    depends_on:
+      - mongodb
+    image: zyner/xyter:2022.07.0
+    restart: unless-stopped
+    environment:
+      - DISCORD_TOKEN=YOUR TOKEN
+      - DISCORD_CLIENT_ID=YOUR CLIENT ID
+      - DISCORD_GUILD_ID=YOUR GUILD ID
+      ### LEAVE THIS AS IT IS! ###
+      - MONGO_URL=mongodb://mongodb:27017/xyter?retryWrites=true&w=majority
+      ############################
+      - ENCRYPTION_ALGORITHM=aes-256-ctr
+      - ENCRYPTION_SECRET=A RANDOM STRING WITH LENGTH OF 32
+      - EMEBD_COLOR_SUCCESS=#22bb33
+      - EMBED_COLOR_WAIT=#f0ad4e
+      - EMBED_COLOR_ERROR=#bb2124
+      - EMBED_FOOTER_TEXT=https://github.com/ZynerOrg/xyter
+      - EMBED_FOOTER_ICON=https://github.com/ZynerOrg.png
+      - LOG_LEVEL=silly
+      - REPUTATION_TIMEOUT=86400
+      - BOT_HOSTER_NAME=Zyner
+      - BOT_HOSTER_URL=https://xyter.zyner.org/customization/change-hoster
+      - NODE_ENV=production
+    stdin_open: true
+    tty: true
+  mongodb:
+    image: mongo:4.2.21
```

Comment:
I think I regret this change, I think it's better to say ``mongo:latest``. To keep security up to date for users. And make them change it themself if they want to.